src/pages/docs/kubernetes/workers/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-01-01
+modDate: 2024-08-14
+title: Kubernetes Workers
+navTitle: Workers
+navSection: Workers
+description: Kubernetes workers
+navOrder: 10
+---
+
+Read up on [Workers](/docs/infrastructure/workers) to understand their role in an Octopus environment, then come back to 
+see how Kubernetes can simplify maintaining them.
+
+Now that you've seen why workers are required, and how to install their various flavours, it's worth describing how the
+Kubernetes specific version can work for you.
+
+For all intents and purposes, the Kubernetes Worker _is_ a standard Octopus Worker, but brings with it unique Kubernetes capabilities
+to ensure hardware utilization scales fluidly with demanded workload.
+
+## Default Behavior
+The vanilla [installation process]((/docs/infrastructure/workers)) installs a worker which will work for 90% of all workloads.
```

Comment:
Can we suggest when customers should consider customisations?

Reply:
There's a few basic pieces below, but its all captured in the kubernetes-agent's readme.